ledger/apply/application_test.go

```diff
@@ -768,18 +839,22 @@ func TestAppCallClearState(t *testing.T) {
 	b.pass = true
 	// check app not exist and not opted in
 	b.balances[sender] = basics.AccountData{}
-	err := ApplicationCall(ac, h, &b, ad, 0, &ep, txnCounter)
+	err := ApplicationCall(ac, h, b, ad, 0, &ep, txnCounter)
 	a.Error(err)
 	a.Contains(err.Error(), "is not currently opted in to app")
 	a.Equal(0, b.put)
+	a.Equal(0, b.putAppLocalState)
+	a.Equal(0, b.deleteAppLocalState)
```

Comment:
I certainly don't care whether you do this or not, but I just learned `a.Zero(b.put)` exists, if you're interested.

Reply:
sure. The same applies to many other of our tests files. I'd rather avoid making a mass-change here, since it's not really related to this PR. But I've fixed all the cases on this file.